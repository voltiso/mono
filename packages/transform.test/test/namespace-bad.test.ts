// â €â“¥ 2022     ðŸŒ©    ðŸŒ©     â €   â €
// â €         ðŸŒ© VÍ›oÍ›Í›Í›ltÍ›Í›Í›iÍ›Í›Í›Í›soÍ›Í›Í›.comâ €  â €â €â €

import * as fs from 'node:fs/promises'

describe('namespace-bad', () => {
	it('works', async () => {
		expect.hasAssertions()

		const file = await fs.readFile('./dist/cjs/namespace-bad.d.ts')
		expect(file.toString()).toMatchSnapshot()
	})
})
