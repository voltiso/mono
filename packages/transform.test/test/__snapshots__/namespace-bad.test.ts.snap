// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`namespace-bad works 1`] = `
"import type { Test } from './_/external-stuff';
export declare type Flatten<T> = [{
    [k in keyof T]: T[k];
}][0];
/** @inline */ export declare type InlineFlatten<T> = [{
    [k in keyof T]: T[k];
}][0];
declare const obj: {
    a: Test.A;
    b: Test.B;
};
export declare type CannotBeInlined = InlineFlatten<typeof obj>;
export {};
//# sourceMappingURL=namespace-bad.d.ts.map"
`;
